### PR TITLE
Fixed: Various Translations

### PR DESCRIPTION
**Pull request by bakerboy448**
#### Database Migration
NO

#### Description
A few sentences describing the overall goals of the pull request's commits.

#### Todos
- [ ] Tests
- [x] Translation Keys (./src/NzbDrone.Core/Localization/Core/en.json)
- [ ] [Wiki Updates](https://wiki.servarr.com)

#### Issues Fixed or Closed by this PR

* Fixes #XXXX

### CODE REVIEW

File: src/NzbDrone.Core/Localization/Core/en.json [hunk 1]
```diff
@@ -668,11 +652,7 @@
   "ThisWillApplyToAllIndexersPleaseFollowTheRulesSetForthByThem": "This will apply to all indexers, please follow the rules set forth by them",
   "Time": "Time",
   "TimeFormat": "Time Format",
-  "Timeleft": "Time Left",
   "Title": "Title",
-  "Titles": "Titles",
-  "Today": "Today",
-  "Tomorrow": "Tomorrow",
```
Comment (RetroDadson) [inline src/NzbDrone.Core/Localization/Core/en.json]: Why deleting these? Are they not used on the calendar page?

File: src/NzbDrone.Core/Localization/Core/en.json [hunk 2]
```diff
@@ -68,18 +65,14 @@
...
   "BookFileCountBookCountTotalTotalBookCountInterp": "{0} / {1} (Total: {2})",
   "BookFileCounttotalBookCountBooksDownloadedInterp": "{0}/{1} books downloaded",
   "BookFilesCountMessage": "No book files",
-  "BookHasNotAired": "Book has not aired",
+  "BookIndex": "Book Index",
   "BookIsDownloading": "Book is downloading",
   "BookIsDownloadingInterp": "Book is downloading - {0}% {1}",
-  "BookIsNotMonitored": "Book is not monitored",
   "BookList": "Book List",
-  "BookMissingFromDisk": "Book missing from disk",
```
Comment (RetroDadson) [inline src/NzbDrone.Core/Localization/Core/en.json:82]: Why deleting?

**Review by RetroDadson — CHANGES_REQUESTED** [review]
There are a lot of deletes, some of which are most certainly used within the UI somewhere (or should be).

File: src/NzbDrone.Core/Localization/Core/en.json (hunk 1 above)
Comment (bakerboy448) [reply src/NzbDrone.Core/Localization/Core/en.json]: Per Nit's script that are not used.

per a search of GitHub they are not used.  "Today" is no where in the UI.

there is "'Today, " which is not translated

https://github.com/Readarr/Readarr/blob/6fa56b88733dd38687401304280a7e40377b58fe/frontend/src/Utilities/Date/formatDateTime.js

File: src/NzbDrone.Core/Localization/Core/en.json (hunk 2 above)
Comment (bakerboy448) [reply src/NzbDrone.Core/Localization/Core/en.json:82]: There is no existing translation for this.

https://github.com/Readarr/Readarr/blob/d87bf5ae63e5e536560c231612e613404376986d/frontend/src/Activity/History/Details/HistoryDetails.js

will need updating

File: frontend/src/System/Tasks/Queued/QueuedTasks.js [hunk 3]
```diff
@@ -10,32 +10,32 @@ import QueuedTaskRowConnector from './QueuedTaskRowConnector';
 const columns = [
   {
     name: 'trigger',
-    label: '',
+    label: translate('Trigger'),
```
Comment (Qstick) [inline frontend/src/System/Tasks/Queued/QueuedTasks.js]: This should always be blank, no translate

Comment (bakerboy448) [reply frontend/src/System/Tasks/Queued/QueuedTasks.js]: squished

**Comment by ta264** [comment]
merged in 94adb4d582f1e14861638ff22e1073db15ecef3e

**Pull request closed without merging**